src/main/java/seedu/moneygowhere/commands/ConsoleCommandSortExpense.java

```diff
@@ -31,10 +28,37 @@ public ConsoleCommandSortExpense(String type, String order) {
         }
     }
 
+    public String getType() {
+        return type;
+    }
+
+    public String getOrder() {
+        if (order) {
+            return ConsoleParserConfigurations.COMMAND_SORT_EXPENSE_ARG_ORDER_VAL_ASCENDING;
+        }
+        return ConsoleParserConfigurations.COMMAND_SORT_EXPENSE_ARG_ORDER_VAL_DESCENDING;
+    }
+
+    /**
+     * Function to get the comparator based on type of sorting.
+     * @return a comparator that corresponds to the type of sorting
+     */
```

Comment:
According to the coding standard, an empty line between description and parameter section is required.